next.config.js

```diff
@@ -227,6 +227,26 @@ module.exports = async (phase) => {
       unoptimized: isStaticBuild,
       domains: imageDomains,
     },
+    async rewrites() {
+      return {
+        beforeFiles: [
+          {
+            source: "/pupils/lessons/:lessonSlug",
+            destination: "/pupils/lessons/:lessonSlug/overview",
+          },
+          {
+            source:
+              "/pupils/programmes/:programmeSlug/units/:unitSlug/lessons/:lessonSlug",
+            destination:
+              "/pupils/programmes/:programmeSlug/units/:unitSlug/lessons/:lessonSlug/overview",
+          },
+          {
+            source: "/pupils/l/:redirectFrom/lessons/:lessonSlug",
+            destination: "/pupils/l/:redirectFrom/lessons/:lessonSlug/overview",
+          },
```

Comment:
Rewrite from the root of the lesson to the overview section — this makes it much easier to manage since we don't need to create a separate page to handle both the root and section routes.